exchange-scrapers/cmd/scraper.go

```diff
@@ -66,6 +67,8 @@ func main() {
 					s[configPair.Exchange] = scrapers.NewCoinBaseScraper()
 				case dia.KrakenExchange:
 					s[configPair.Exchange] = scrapers.NewKrakenScraper(configApi.ApiKey, configApi.SecretKey)
+				case dia.HitBTCExchange:
+					s[configPair.Exchange] = scrapers.NewHitBTCScraper(configApi.ApiKey, configApi.SecretKey)
```

Comment:
Hi PumkpingWok, why using an api key and secrekey? you're not using it later.

Reply:
Hi @frranck,
I have left these two parameters if in the future someone would implement a client other than connection to websocket, i can change this without problems.